adapter/src/adapter/model/exceptions/AdapterError.ts

```diff
@@ -0,0 +1,5 @@
+export class AdapterError extends Error {
```

Comment:
Dunno if we need that in typescript?

Reply:
These are used for catching certain errors and returning a different response, e.g.:
https://github.com/jvalue/ods/blob/4fe866fdb2228525e80e6414c946785166e7ff4a/adapter/src/datasource/api/amqp/amqpConsumer.ts#L102-L125